docs/configure.rst

```diff
@@ -42,3 +42,31 @@ Example:
 .. sourcecode:: python
 
     mysql_proc = factories.mysql_proc(port=3308, logs_prefix='myproject-')
+
+
+How the executable files are searched  
+=====================================
+
+Configuration file contain information where to find the executables of each supported database and storage engine.
+Executables may be indicated with (or without) an absolute path, for instance:
+
+.. sourcecode:: python
+
+    example_exec: /usr/local/bin/example
```

Comment:
I don't think dbfixtures is that much generic Ther's subkey redis_exec, but there are also deamon for elasicsearch, mysql_init, mysql_server and mysql_client for mysql executables